src/scripts/items/buyKeyItem.ts

```diff
@@ -0,0 +1,22 @@
+class buyKeyItem extends Item {
```

Comment:
Maybe we can merge this class with the current key item class

Reply:
Wouldn't this force all key items to have a price, totalPrice etc, which is going to be useless for most of them?